libraries/AP_DDS/README.md

```diff
@@ -219,6 +219,10 @@ ros2 topic echo /ap/tf_static --qos-depth 1 --qos-history keep_last --qos-reliab
 
 In order to consume the transforms, it's highly recommended to [create and run a transform broadcaster in ROS 2](https://docs.ros.org/en/humble/Concepts/About-Tf2.html#tutorials).
 
+
+## ROS 2 Subscriptions
```

Comment:
There is no subscription, I think the above section is fine by itself to explain that ArduPilot publishes this. 
Detailed API docs like what the frame ID's and coordinates should go in the wiki.

Reply:
Ok, I will remove this section.